lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -140,7 +142,7 @@ sub cache_assets {
             $status = $cache_client->status($asset_request);
         }
         my $msg = "Download of $asset_uri processed";
-        if (my $output = $status->output) { $msg .= ":\n$output" }
+        if (my $output = $status->output) { $msg .= ":\n" . indent_log_raw_output($output) }
```

Comment:
I'm wondering why the first line shouldn't be indented here.
After all the return value of that function is already going to be a continuation line of the log message, so *all* lines should be indented by that function.

The description of the PR should include a before and after example.